tests/gdk/build_system/test_zip_build_configuraiton.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+from faker import Faker
+from faker.providers import color
+import pytest
+from unittest import TestCase
+
+from gdk.build_system.Zip import Zip
+from gdk.common.configuration import validate_configuration
+
+
+class ZipConfigurationTests(TestCase):
+
+    def configuration_base(self, options: dict):
+        base = {
+            "component": {
+                "com.build.zip": {
+                    "author": "abc",
+                    "version": "1.0.0",
+                    "build": {
+                        "build_system": "zip"
+                    },
+                    "publish": {
+                        "bucket": "default",
+                        "region": "us-east-1"
+                    },
+                }
+            },
+            "gdk_version": "1.0.0",
+        }
+
+        if options is not None:
+            base["component"]["com.build.zip"]["build"].setdefault("options", options)
+
+        return base
+
+    def test_valid_configuration_options(self):
```

Comment:
Nit: You can use `pytest.mark.parametrize` for passing multiple inputs in these two tests.